packages/immutadot/src/core/flow.spec.js

```diff
@@ -115,4 +116,21 @@ describe('flow.flow', () => {
       other: {},
     })
   })
+  it('should update two different array indexes', () => {
```

Comment:
Here is the failing test.
The expected output is OK, but the `'nested.prop.1.val'` is mutated in the original object !